cmake/patches/onnx/onnx.patch

```diff
@@ -1,16 +1,16 @@
-diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Note: I created this patch by:
- Checking out tag v1.16.0 in a local onnx repo
- applying the old patch from onnxruntime/cmake/patches/onnx/onnx.patch
- Applying changes from https://github.com/onnx/onnx/pull/6080
- Creating a new patch: `git diff > onnx.patch`

Reply:
If there's a better way to generate the patch that minimizes the diff, please let me know